File: SecondYear/Adityacheeze_AdityaMohanGupta_2226CSE(AI&ML)1018_2/Week_2/Fibbonacci.cpp
vector<int> generateFibonacciNumbers(int n) {
    // Write your code here.
    vector<int>res(n);
    res[1]=1;
    for(int i=2; i<n ;i++){
        res[i] = res[i-1] + res[i-2];
    }
    return res;
}

File: SecondYear/gamandeepsingh_GamandeepSingh_2226csit1042_2/Week_2/Fibbonacci.java
public class Solution {
    public static int[] generateFibonacciNumbers(int n) {
        // Write your code here.
        int[] fib = new int[n];
        fib[0]=0;
        if(n==1){
            return fib; 
        }
        fib[1]=1;
        for(int i=2;i<n;i++){
            fib[i]=fib[i-1]+fib[i-2];
        }
        return fib;

    }
}